src/System.Net.Security/tests/FunctionalTests/TestConfiguration.cs

```diff
@@ -14,7 +14,7 @@ namespace System.Net.Security.Tests
 {
     internal static class TestConfiguration
     {
-        public const int PassingTestTimeoutMilliseconds = 1 * 60 * 1000;
+        public const int PassingTestTimeoutMilliseconds = 4 * 60 * 1000;
```

Comment:
Are they hitting the timeout legitimately, or are they hanging and would hit any timeout?  60 seconds is already a very long timeout; why do we believe they didn't finish in 1 minute but will finish in 4?